### PR TITLE
[WIFI-1724] SSC reconnect to cassandra after cassandra pod restart

### PR DESCRIPTION
**Pull request by norm-traxler**

### CODE REVIEW

**Review by AkshayJagadish-ne — APPROVED** [review]

File: tip-wlan/charts/wlan-ssc-service/templates/configmap.yaml [hunk 1]
```diff
@@ -11,8 +11,10 @@ data:
             contact-points = [ "{{ include "cassandra.service" . }}:9042" ]
             load-balancing-policy.local-datacenter = datacenter1
             session-keyspace = tip_wlan_keyspace
+            request.consistency = LOCAL_QUORUM
```
Comment (brennerm) [inline tip-wlan/charts/wlan-ssc-service/templates/configmap.yaml:14]: Why is this change required?

**Review by brennerm — COMMENTED** [review]
Reconnecting works fine with the `advanced.resolve-contact-points = false` configuration. Don't see the need for adding `request.consistency = LOCAL_QUORUM` though.

File: tip-wlan/charts/wlan-ssc-service/templates/configmap.yaml (hunk 1 above)
Comment (norm-traxler) [reply tip-wlan/charts/wlan-ssc-service/templates/configmap.yaml:14]: For a deployment with a single replica Cassandra, this setting doesn't matter. But if you were to deploy 3 replicas of Cassandra, then using "LOCAL_QUORUM" will allow ssc to continue to function with the other 2 available Cassandras when cassandra-0 goes down.

**Review by brennerm — APPROVED** [review]

**Comment by brennerm** [comment]
@norm-traxler Please add an entry to the CHANGELOG.md file that reflects this change.

**Review by yongchen-cu — APPROVED** [review]
